### PR TITLE
DBT-1337 -> CSS Theme: Added a CSS theme that holds global CSS values for colors, sizes and many other css properties.

### PR DESCRIPTION
**Pull request by deevannaveed**
## Context
The purpose of this PR is to add a global CSS theme with colors, sizes and other basic CSS properties, this would allow us the control colors, sizes and other css properties from one places and it will also make it easier to add more themes as needed.

**Note:** This PR is still a work in progress, this some improvements.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (deevannaveed) [inline pnpm-lock.yaml:1]: Not sure why this file is always updated when I pull new changes! but this basically happens when local changes in to this file during a feature development are not pushed in the PR.

Comment (abelbordonado) [reply pnpm-lock.yaml:1]: PNPM is misterious.
I don;t know how to solve conflicts well.

Bext way is to remove it and regenerated with `pnpm install` if there is a doubt

File: pages/globals.css [hunk 1]
```diff
@@ -3,16 +3,203 @@
...
+  --gray-50: #e9f3fe
+}
+:root {
+  --radii-none: 0;
+  --radii-sm: 0.125rem;
+  --radii-md: 0.25rem;
+  --radii-lg: 0.5rem;
+  --radii-full: 9999px
+}
+:root {
```
Comment (abelbordonado) [inline pages/globals.css:114]: We already have tailwindcss for some of this style, but I think doesn't hurt.

File: pages/globals.css [hunk 2]
```diff
@@ -3,16 +3,203 @@
...
+  --sticky: 1100;
+  --banner: 1200;
+  --overlay: 1300;
+  --modal: 1400;
+  --popover: 1500;
+  --skipLink: 1600;
+  --toast: 1700;
+  --tooltip: 180
+}
+:root {
```
Comment (abelbordonado) [inline pages/globals.css:190]: This is the important yes :D @Mrtn-Omwenga

File: pages/globals.css (hunk 1 above)
Comment (deevannaveed) [reply pages/globals.css:114]: Yeah, Tailwindcss actually gives us classes which cannot be used as css properties with custom css.

File: pnpm-lock.yaml
Comment (deevannaveed) [reply pnpm-lock.yaml:1]: Actually I did `pnpm install` which should install the packages that has the same version as in the `pnpm-lock.yml` but in my case it  changed a lot in the `pnpm-lock.yml`, anyways its not breaking something now but I think it should be stable.

**Pull request merged**